dwave/cloud/cli.py

```diff
@@ -1051,6 +1051,32 @@ def refresh(*, config_file, profile, output):
            'You can now use "dwave auth get" to view them.')
 
 
+@auth.command()
+@config_file_options()
+@click.argument('token_type', default='access-token',
+                type=click.Choice(['access-token', 'refresh-token']))
+@json_output
+@standardized_output
+def revoke(*, config_file, profile, token_type, json_output, output):
+    """Revoke Leap API access and/or refresh token(s)."""
+
+    config = validate_config_v1(load_config(config_file=config_file, profile=profile))
+    flow = LeapAuthFlow.from_config_model(config)
+
+    token_key = token_type.replace('-', '_')
+    token_pretty = token_type.replace('-', ' ').capitalize()
+    if not flow.token or not (token_value := flow.token.get(token_key)):
```

Comment:
Not to be Guido, but IMO if you're going to use `token_value` outside of the if block I would not use the walrus operator here. Took me a moment to find where it got assigned. I guess it might make sense if `flow.token.get(...)` is very expensive and you want to skip it if `not flow.token`, but IMO once you're throwing errors performance isn't as important.

Reply:
Agree, I'll fix it. The code looked differently when I wrote it like this.